tests/e2e/transitions/sms_workflows.spec.js

```diff
@@ -98,14 +99,26 @@ const expectTasks = (doc, expectations) => {
     .to.have.deep.members(expectations);
 };
 
+const processReportsAndSetings = async (reports, settings) => {
+  const ids = reports.map(report => report._id);
+  await utils.updateSettingsNative(settings);
+  await utils.saveDocsNative(reports);
+  await sentinelUtils.waitForSentinelNative(ids);
+  return utils.getDocsNative(ids);
+};
+
 describe('SMS workflows', () => {
-  beforeAll(() => utils.saveDocs(contacts));
-  afterAll(() => utils.revertDb());
+  beforeAll(async () => { 
+    commonPo.goToPeople();
```

Comment:
Why do you need to go to the people page here? 
Without it, similar to Craig's comment below, this can become: 
```
beforeAll(() => utils.saveDocsNative(contacts));
```